nexus/src/external_api/http_entrypoints.rs

```diff
@@ -651,7 +651,7 @@ async fn silo_quotas_view(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-/// Update the resource quotas of a given silo
+/// Update resource quotas for silo
```

Comment:
Was going to suggest getting this smaller with: "Update silo resource quotas" – but probably less readable.